### PR TITLE
[AC][Smartlab] Action recognition and object detection models and accuracy tools

### PR DESCRIPTION
**Pull request by kminemur**
need accuracy tool implementation.
our reference accuracy tools: https://github.com/kminemur/smartlab_accracy_test

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by vladimir-dudnik** [comment]
please review precommit checks failed
16:44:25  models/intel/smartlab-object-detection-0003/accuracy-check.yml:7: trailing whitespace.
16:45:13  models/intel/smartlab-object-detection-0001/accuracy-check.yml:35: last line doesn't end with a newline character

**Comment by kminemur** [comment]
solved issues.
now passing all

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Example of the output
+
+`openvino`
+
+## Composite model specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the DSI1867                        | TODO               |
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:20]: DSI1867 is internal dataset name, it should not be a part of public documentation
```suggestion
| Accuracy (internal test set)                        | TODO               |
```

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+| Metric  | Value |
+| ------- | ----- |
+| GFlops  | 0.008 |
+| MParams | 4.099 |
+
+### Inputs
+
+1.	Name: `input_feature_1`, shape: `1, 1280`. Encoded features from topview.
+2.	Name: `input_feature_1`, shape: `1, 1280`. Encoded features from frontview.
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:58]: why 2 inputs has the same name?

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+The smartlab-action-recognition-encoder-0001 is a Mobilenet-V2 like backbone with convolutional encoder part of the action recognition.
+
+| Metric  | Value |
+| ------- | ----- |
+| GFlops  | 0.611 |
+| MParams | 3.387 |
+
+### Inputs
+
+Image, name: `input_image`, shape: `1, 1, 224, 224` in the `1, C, H, W` format, where:
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:35]: are you sure that model has single channel input? In internal repo I see 3 channels
```suggestion
Image, name: `input_image`, shape: `1, 3, 224, 224` in the `N, C, H, W` format, where:
```

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:39]:
```suggestion

- `B` - batch size
- `C` - number of channels
- `H` - image height
- `W` - image width
```

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### Outputs
+
+1.	Name: `output_feature`, shape: `1, 1280`. Features from encoder part of action recogntion head.
+
+## Decoder model specification
+
+The smartlab-action-recognition-decoder-0001 is a fully connected layer module.
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:48]:
```suggestion
The smartlab-action-recognition-decoder-0001 is a fully connected decoder part which accepts features from top and front views, computed by encoder and predicts score for action across following label list: `no_action`, `noise_action`,  `adjust_rider`
```

File: models/intel/smartlab-action-recognition-0001/composite-model.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+description: >-
+  General-purpose action recognition model for DSI1867 dataset
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/composite-model.yml:2]: please remove all mentions of internal dataset ID!!!

File: models/intel/smartlab-object-detection-0001/README.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
+| GFlops                          | 1.077                                     |
+| MParams                         | 0.8908                                    |
+| Source framework                | PyTorch\*                                 |
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
```
Comment (eaidova) [inline models/intel/smartlab-object-detection-0001/README.md:24]:
```suggestion
Image, name: `images`, shape: `1, 3, 416, 416` in the format `B, C, H, W`, where:
```

File: models/intel/smartlab-object-detection-0001/README.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Outputs
+
+The net outputs blob with shape: `1, 1, 200, 7` in the format `1, 1, N, 7`, where `N` is the number of detected
```
Comment (eaidova) [inline models/intel/smartlab-object-detection-0001/README.md:35]: it is not yolox format of output and does not have any relation to your model output
If you do not know, how to describe model output format, please use this template 
https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/yolox-tiny#converted-model-1

File: models/intel/smartlab-object-detection-0001/accuracy-check.yml [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: yolox
+          anchors: yolox
+          classes: 10
+
+    datasets:
+      - name: DSI1867
```
Comment (eaidova) [inline models/intel/smartlab-object-detection-0001/accuracy-check.yml:13]: you should not mention internal dataset id in any public files, please gived something neutral

File: models/intel/smartlab-object-detection-0001/accuracy-check.yml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        data_source: ..\..\..\DSI-1867\object_detection\streams_1\top\images
```
Comment (eaidova) [inline models/intel/smartlab-object-detection-0001/accuracy-check.yml:14]: I mean differnet when says relative path.
AC has command line arguments -s, --source which stores the root of datasets directory. Path in configs should be written relative to it without and special symbols like `..`, `.`, `~`

Comment (eaidova) [reply models/intel/smartlab-object-detection-0001/accuracy-check.yml:14]: beside that data_source, annotation, annotation_conversion should be moved to dataset_definitions file, they should not be defined in model config
https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_definitions.yml

**Comment by eaidova** [comment]
@kminemur, please apply comments regarding configs and input/output formatting in readme to all relevant files (I just do not want repeat these comments for every file)

File: models/intel/smartlab-action-recognition-0001/accuracy-check.yml [hunk 11]
```diff
@@ -0,0 +1,34 @@
+evaluations:
+  - name: smartlab-action-recognition-0001
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/accuracy-check.yml:2]: please remove this file, I will provide my own config in separated request

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+### Outputs
+
+1.	Name: `decoder_hidden`, shape: `1, 3`. The foramt [`has_action_conf_score`, `action_1_logits`, `action_2_logits`]
+
+- `has_action_conf_score` - confidence for action frame
+- `noise_action` - confidence for the noise_action class
+- `adjust_rider` - confidence for the adjust_rider class
+
+Classification confidence scores in the [0, 1] range
+    for every smartlab actions.
```
Comment (eaidova) [inline models/intel/smartlab-action-recognition-0001/README.md:69]:
```suggestion
Classification confidence scores in the [0, 1] range
```

File: models/intel/smartlab-action-recognition-0001/README.md (hunk 2 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/README.md]: fixed

File: models/intel/smartlab-action-recognition-0001/README.md (hunk 3 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/README.md]: fixed

File: models/intel/smartlab-action-recognition-0001/README.md (hunk 5 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/README.md]: fixed

File: models/intel/smartlab-action-recognition-0001/composite-model.yml (hunk 6 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/composite-model.yml]: fixed

File: models/intel/smartlab-object-detection-0001/README.md (hunk 7 above)
Comment (kminemur) [reply models/intel/smartlab-object-detection-0001/README.md]: fixed

File: models/intel/smartlab-object-detection-0001/README.md (hunk 8 above)
Comment (kminemur) [reply models/intel/smartlab-object-detection-0001/README.md]: thanks.

fixed the description

File: models/intel/smartlab-object-detection-0001/accuracy-check.yml (hunk 9 above)
Comment (kminemur) [reply models/intel/smartlab-object-detection-0001/accuracy-check.yml]: changed all of DSI**** name

File: models/intel/smartlab-object-detection-0001/accuracy-check.yml (hunk 10 above)
Comment (kminemur) [reply models/intel/smartlab-object-detection-0001/accuracy-check.yml]: fixed

File: models/intel/smartlab-object-detection-0002/accuracy-check.yml [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+        device: CPU
+        adapter:
+          type: yolox
+          anchors: yolox
+          classes: 3
+
+    datasets:
+      - name: DSI1867
+        data_source: ..\..\..\DSI-1867\object_detection\streams_1\top\images
+        annotation: dir_conv_top\mscoco_detection_top_3cls.pickle
```
[the remark this answers is not in the document]
Comment (kminemur) [reply models/intel/smartlab-object-detection-0002/accuracy-check.yml]: fixed

File: models/intel/smartlab-action-recognition-0001/accuracy-check.yml (hunk 11 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/accuracy-check.yml]: removed

File: models/intel/smartlab-action-recognition-0001/README.md (hunk 12 above)
Comment (kminemur) [reply models/intel/smartlab-action-recognition-0001/README.md]: fixed

File: models/intel/smartlab-action-recognition-0001/README.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy (internal test set)                   | TODO               |
+| Source framework                               | PyTorch\*          |
+
+
+## Encoder model specification
+with convolutional encoder part of the action re
+The smartlab-action-recognition-encoder-0001 is a Mobilenet-V2 like backbone cognition.
```
Comment (Anna-Mironova) [inline models/intel/smartlab-action-recognition-0001/README.md:26]: Possibly it's a misprint
```suggestion
The smartlab-action-recognition-encoder-0001 is a Mobilenet-V2 like backbone with convolutional encoder part of the action recognition.
```

File: models/intel/smartlab-object-detection-0001/README.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+- B - batch size
+- N - number of detection boxes
+
+Detection box has format [x, y, h, w, box_score, class_no_1, ..., class_no_10], where:
+
+- (`x`, `y`) - raw coordinates of box center
+- `h`, `w` - raw height and width of box
+- `box_score` - confidence of detection box
+- `class_no_1, ..., class_no_10` - probability distribution over the classes in logits format.
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0001/README.md:45]:
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 10`, format is `B, N, 10`, where:

- `B` - batch size
- `N` - number of detection boxes

Detection box has format [`x`, `y`, `h`, `w`, `box_score`, `class_no_1`, ..., `class_no_10`], where:

- (`x`, `y`) - raw coordinates of box center
- `h`, `w` - raw height and width of box
- `box_score` - confidence of detection box
- `class_no_1`, ..., `class_no_10` - probability distribution over the classes in logits format.
```

File: models/intel/smartlab-object-detection-0002/README.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+- B - batch size
+- N - number of detection boxes
+
+Detection box has format [x, y, h, w, box_score, class_no_1, ..., class_no_3], where:
+
+- (`x`, `y`) - raw coordinates of box center
+- `h`, `w` - raw height and width of box
+- `box_score` - confidence of detection box
+- `class_no_1, ..., class_no_3` - probability distribution over the classes in logits format.
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0002/README.md:44]:
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 3`, format is `B, N, 3`, where:

- `B` - batch size
- `N` - number of detection boxes

Detection box has format [`x`, `y`, `h`, `w`, `box_score`, `class_no_1`, ..., `class_no_3`], where:

- (`x`, `y`) - raw coordinates of box center
- `h`, `w` - raw height and width of box
- `box_score` - confidence of detection box
- `class_no_1`, ..., `class_no_3` - probability distribution over the classes in logits format.
```

File: models/intel/smartlab-object-detection-0003/README.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+
+- B - batch size
+- N - number of detection boxes
+
+Detection box has format [x, y, h, w, box_score, class_no_1, ..., class_no_10], where:
+
+- (`x`, `y`) - raw coordinates of box center
+- `h`, `w` - raw height and width of box
+- `box_score` - confidence of detection box
+- `class_no_1, ..., class_no_10` - probability distribution over the classes in logits format.
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0003/README.md:45]:
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 10`, format is `B, N, 10`, where:

- `B` - batch size
- `N` - number of detection boxes

Detection box has format [`x`, `y`, `h`, `w`, `box_score`, `class_no_1`, ..., `class_no_10`], where:

- (`x`, `y`) - raw coordinates of box center
- `h`, `w` - raw height and width of box
- `box_score` - confidence of detection box
- `class_no_1`, ..., `class_no_10` - probability distribution over the classes in logits format.
```

File: models/intel/smartlab-object-detection-0004/README.md [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+
+- B - batch size
+- N - number of detection boxes
+
+Detection box has format [x, y, h, w, box_score, class_no_1, ..., class_no_3], where:
+
+- (`x`, `y`) - raw coordinates of box center
+- `h`, `w` - raw height and width of box
+- `box_score` - confidence of detection box
+- `class_no_1, ..., class_no_3` - probability distribution over the classes in logits format.
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0004/README.md:49]:
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 3`, format is `B, N, 3`, where:

- `B` - batch size
- `N` - number of detection boxes

Detection box has format [`x`, `y`, `h`, `w`, `box_score`, `class_no_1`, ..., `class_no_3`], where:

- (`x`, `y`) - raw coordinates of box center
- `h`, `w` - raw height and width of box
- `box_score` - confidence of detection box
- `class_no_1`, ..., `class_no_3` - probability distribution over the classes in logits format.
```

File: models/intel/smartlab-object-detection-0002/README.md [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+
+| Metric                          | Value                                     |
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
+| GFlops                          | 1.073                                     |
+| MParams                         | 0.8894                                    |
+| Source framework                | PyTorch\*                                 |
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0002/README.md:23]: Possibly it's a misprint
```suggestion
Image, name: `images`, shape: `1, 3, 416, 416` in the format `B, C, H, W`, where:
```

File: models/intel/smartlab-object-detection-0003/README.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
+| GFlops                          | 1.077                                     |
+| MParams                         | 0.8908                                    |
+| Source framework                | PyTorch\*                                 |
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0003/README.md:24]:
```suggestion
Image, name: `images`, shape: `1, 3, 416, 416` in the format `B, C, H, W`, where:
```

File: models/intel/smartlab-object-detection-0004/README.md [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+| MParams                         | 0.8894                                    |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0004/README.md:28]:
```suggestion
Image, name: `images`, shape: `1, 3, 416, 416` in the format `B, C, H, W`, where:
```

File: models/intel/smartlab-object-detection-0001/README.md [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Outputs
+
+The array of detection summary info, name - `output`, shape - `1, 3549, 10`, format is `B, N, 10`, where:
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0001/README.md:35]: 10 classes + 4 coordinates + box score
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 15`, format is `B, N, 15`, where:
```

File: models/intel/smartlab-object-detection-0001/README.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Outputs
+
+The array of detection summary info, name - `output`, shape - `1, 3549, 10`, format is `B, N, 15`, where:
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0001/README.md:35]:
```suggestion
The array of detection summary info, name - `output`, shape - `1, 3549, 15`, format is `B, N, 15`, where:
```

File: models/intel/smartlab-object-detection-0001/accuracy-check.yml [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+
+    datasets:
+      - name: smartlab
+        data_source: object_detection\streams_1\top\images
+
+        annotation_conversion:
+          converter: mscoco_detection
+          annotation_file: object_detection\streams_1\top\annotations\instances_glb1cls10.json
+        annotation: mscoco_detection_top_10cls.pickle
+        dataset_meta: mscoco_detection_top_10cls.json
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0001/accuracy-check.yml:19]:
```suggestion
      - name: smartlab_detection_10cl_top
```

File: models/intel/smartlab-object-detection-0002/accuracy-check.yml [hunk 25]
```diff
@@ -0,0 +1,38 @@
...
+
+    datasets:
+      - name: smartlab
+        data_source: object_detection\streams_1\top\images
+
+        annotation_conversion:
+          converter: mscoco_detection
+          annotation_file: object_detection\streams_1\top\annotations\instances_glb2bcls3.json
+        annotation: mscoco_detection_top_3cls.pickle
+        dataset_meta: mscoco_detection_top_3cls.json
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0002/accuracy-check.yml:19]:
```suggestion
      - name: smartlab_detection_3cl_top
```

File: models/intel/smartlab-object-detection-0003/accuracy-check.yml [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+          classes: 10
+
+    datasets:
+      - name: smartlab
+        data_source: object_detection\streams_1\high\images
+        annotation_conversion:
+          converter: mscoco_detection
+          annotation_file: object_detection\streams_1\high\annotations\instances_glb1cls10.json
+        annotation: mscoco_detection_high_10cls.pickle
+        dataset_meta: mscoco_detection_high_10cls.json
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0003/accuracy-check.yml:18]:
```suggestion
      - name: smartlab_detection_10cl_front
```

File: models/intel/smartlab-object-detection-0004/accuracy-check.yml [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+
+    datasets:
+      - name: smartlab
+        data_source: object_detection\streams_1\high\images
+        annotation_conversion:
+          converter: mscoco_detection
+          annotation_file: object_detection\streams_1\high\annotations\instances_glb2bcls3.json
+        annotation: mscoco_detection_high_3cls.pickle
+        dataset_meta: mscoco_detection_high_3cls.json
+
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0004/accuracy-check.yml:19]:
```suggestion
      - name: smartlab_detection_3cl_front
```

**Comment by kminemur** [comment]
anything left to work?

File: data/dataset_definitions.yml [hunk 28]
```diff
@@ -1461,3 +1461,35 @@ datasets:
       data_dir: annotation
       input_suffix: in
       reference_suffix: out
+
+  - name: smartlab_detection_10cl_top
+    data_source: object_detection\streams_1\top\images
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: object_detection\streams_1\top\annotations\instances_glb1cls10.json
```
Comment (Anna-Mironova) [inline data/dataset_definitions.yml:1469]:
```suggestion
    data_source: object_detection/streams_1/top/images
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: object_detection/streams_1/top/annotations/instances_glb1cls10.json
```

File: data/dataset_definitions.yml [hunk 29]
```diff
@@ -1461,3 +1461,35 @@ datasets:
...
+    annotation: mscoco_detection_top_10cls.pickle
+    dataset_meta: mscoco_detection_top_10cls.json
+
+  - name: smartlab_detection_3cl_top
+    data_source: object_detection\streams_1\top\images
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: object_detection\streams_1\top\annotations\instances_glb2bcls3.json
```
Comment (Anna-Mironova) [inline data/dataset_definitions.yml:1477]:
```suggestion
    data_source: object_detection/streams_1/top/images
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: object_detection/streams_1/top/annotations/instances_glb2bcls3.json
```

File: data/dataset_definitions.yml [hunk 30]
```diff
@@ -1461,3 +1461,35 @@ datasets:
...
+    annotation: mscoco_detection_top_3cls.pickle
+    dataset_meta: mscoco_detection_top_3cls.json
+
+  - name: smartlab_detection_10cl_high
+    data_source: object_detection\streams_1\high\images
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: object_detection\streams_1\high\annotations\instances_glb1cls10.json
```
Comment (Anna-Mironova) [inline data/dataset_definitions.yml:1485]:
```suggestion
    data_source: object_detection/streams_1/high/images
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: object_detection/streams_1/high/annotations/instances_glb1cls10.json
```

File: data/dataset_definitions.yml [hunk 31]
```diff
@@ -1461,3 +1461,35 @@ datasets:
...
+    annotation: mscoco_detection_high_10cls.pickle
+    dataset_meta: mscoco_detection_high_10cls.json
+
+  - name: smartlab_detection_3cl_high
+    data_source: object_detection\streams_1\high\images
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: object_detection\streams_1\high\annotations\instances_glb2bcls3.json
```
Comment (Anna-Mironova) [inline data/dataset_definitions.yml:1493]:
```suggestion
    data_source: object_detection/streams_1/high/images
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: object_detection/streams_1/high/annotations/instances_glb2bcls3.json
```

File: models/intel/smartlab-object-detection-0002/README.md [hunk 32]
```diff
@@ -0,0 +1,48 @@
...
+## Example
+
+![](./assets/frame0001_top2.jpg)
+
+## Specification
+
+Accuracy metrics obtained on Smartlab validation dataset with yolox adapter for converted model.
+
+| Metric                          | Value                                     |
+| [COCO mAP (0.5:0.05:0.95)]      | 0.060%                                    |
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0002/README.md:16]:
```suggestion
| [COCO mAP (0.5:0.05:0.95)]      | 6.06%                                    |
```

File: models/intel/smartlab-object-detection-0002/accuracy-check.yml [hunk 33]
```diff
@@ -0,0 +1,31 @@
...
+            overlap: 0.2
+          - type: clip_boxes
+            apply_to: prediction
+
+        metrics:
+          - name: AP@0.5:0.05:95
+            type: coco_precision
+            max_detections: 100
+            threshold: '0.5:0.05:0.95'
+            reference: 0.060
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0002/accuracy-check.yml:31]:
```suggestion
            reference: 0.0606
```

File: models/intel/smartlab-object-detection-0003/README.md [hunk 34]
```diff
@@ -0,0 +1,49 @@
...
+
+![](./assets/frame0001_front1.jpg)
+
+## Specification
+
+Accuracy metrics obtained on Smartlab validation dataset with yolox adapter for converted model.
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0003/README.md:17]:
```suggestion
| [COCO mAP (0.5:0.05:0.95)]      | 30.38%                                    |
```

File: models/intel/smartlab-object-detection-0004/README.md [hunk 35]
```diff
@@ -0,0 +1,53 @@
...
+
+![](./assets/frame0001_front2.jpg)
+
+## Specification
+
+Accuracy metrics obtained on Smartlab validation dataset with yolox adapter for converted model.
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
```
Comment (Anna-Mironova) [inline models/intel/smartlab-object-detection-0004/README.md:17]:
```suggestion
| [COCO mAP (0.5:0.05:0.95)]      | 11.18%                                    |
```

**Review by Anna-Mironova — APPROVED** [review]

**Pull request merged**